src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -630,25 +634,46 @@ internal PhysicalLocation CreatePhysicalLocation(PhysicalLocationVersionOne v1Ph
                 physicalLocation = new PhysicalLocation
                 {
                     FileLocation = CreateFileLocation(v1PhysicalLocation),
-                    Region = CreateRegion(v1PhysicalLocation.Region)
+                    Region = CreateRegion(v1PhysicalLocation)
                 };
             }
 
             return physicalLocation;
         }
 
-        internal Region CreateRegion(RegionVersionOne v1Region)
+        internal Region CreateRegion(PhysicalLocationVersionOne v1PhysicalLocation)
```

Comment:
This is fine. You can reduce coupling by passing `Region region, string uri` (that is, just the bits of the `v1PhysicalLocation` that you need.

Reply:
Agreed, changed.